src/providers/storage/s3.js

```diff
@@ -5,8 +5,10 @@ const s3 = (formio) => ({
       response.data.fileName = fileName;
       response.data.key = XHR.path([response.data.key, dir, fileName]);
       if (response.signed) {
-        xhr.openAndSetHeaders('PUT', response.signed);
```

Comment:
How did this ever work if it used to pass in an OBJECT for the PUT request?

Reply:
You're correct this needed to be updated based on a PR change request on [formio-server](https://github.com/formio/formio-server/pull/1336#discussion_r1333648380) Brendan and I recently discussed, as a result formio.js needed to be updated. 

It was not updated. Have applied requested change as a result of that discussion.
![image](https://github.com/formio/formio.js/assets/112976114/a258fad3-a901-40d5-9272-2ee31ca645b9)